### PR TITLE
AB#1750 - Brand template form

### PR DESCRIPTION
**Pull request by sgaddameedi-insight**
# Description

Brand Template with form fields and store

Fixes #1750

## Type of change

1. File Input for brand logo
2. Agency form fields
3. store for brand
4. WIP: fetch settings API and do `put` to update the settings

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] API unit tests
- [x] UI unit tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jeremiahbooth-insight — APPROVED** [review]
We will need to add a function to the app that if they have set the brand info to set the vuetify color theme to the primary and secondary colors at some point.

File: UI/libs/core-admin/src/lib/components/brand/BrandForm.vue [hunk 1]
```diff
@@ -1,51 +1,201 @@
...
+              </v-col>
+            </v-row>
+            <v-row>
+              <v-col
+                cols="12"
+                md="6"
+              >
+                <v-text-field
+                  :label="$t('Primary Theme Color')"
+                  :rules="[v => !!v || 'Primary Theme color is required']"
```
Comment (jacobkellas) [inline UI/libs/core-admin/src/lib/components/brand/BrandForm.vue:95]: Might want to make these default to Vuetify's default colors if the agency doesn't have a primary or secondary color palette defined.  I'm not sure how many have color standards for applications.

Comment (sgaddameedi-insight) [reply UI/libs/core-admin/src/lib/components/brand/BrandForm.vue:95]: Done. Currently hardcoded them but we can get from vuetufy global config if needed.

**Comment by sgaddameedi-insight** [comment]
> We will need to add a function to the app that if they have set the brand info to set the vuetify color theme to the primary and secondary colors at some point.

@jeremiahbooth-insight  Yes - i got that in the following PR which will basically read from settings API @ofeliaporumb-insight  created and updated vuetify plugin theme before loading the app.

File: UI/libs/core-admin/src/lib/components/brand/BrandForm.vue (hunk 1 above)
Comment (sgaddameedi-insight) [reply UI/libs/core-admin/src/lib/components/brand/BrandForm.vue:95]: ok - found a way to read from vuetify plugins. Updated!

**Pull request merged**